Index: src/main/java/org/intellij/plugin/mdx/lang/psi/MdxTemplateDataElementType.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/org/intellij/plugin/mdx/lang/psi/MdxTemplateDataElementType.kt	(revision 50e1a032215d21018ed4d72a34fc83fdc9b6974c)
+++ src/main/java/org/intellij/plugin/mdx/lang/psi/MdxTemplateDataElementType.kt	(date 1597232702136)
@@ -1,5 +1,6 @@
 package org.intellij.plugin.mdx.lang.psi

+import com.intellij.lang.javascript.types.JEEmbeddedBlockElementType
 import com.intellij.psi.templateLanguages.TemplateDataElementType
 import com.intellij.psi.tree.IElementType
 import com.intellij.psi.tree.TokenSet
@@ -10,7 +11,10 @@
 object MdxTemplateDataElementType : TemplateDataElementType("MDX_TEMPLATE_JSX",
         MdxLanguage,
         MarkdownElementType.platformType(MdxTokenTypes.JSX_BLOCK_CONTENT),
-        IElementType("OUTER_BLOCK", MdxLanguage)) {
+        IElementType("OUTER_BLOCK", MdxLanguage)),
+        JEEmbeddedBlockElementType {
+
+    override fun isModule(): Boolean = true

     override fun getTemplateDataInsertionTokens(): TokenSet {
         return TokenSet.forAllMatching(IElementType.TRUE)